deploy/playbook_bionic.yml

```diff
@@ -100,6 +101,9 @@
     - name: change npm version
       shell: "npm install -g npm@{{npm_version}}" # TODO: idempotent
 
+    - name: ensure system is using PHP 7.3
+      shell: "sudo update-alternatives --set php /usr/bin/php7.3"
```

Comment:
I think this should be /usr/bin/php{{php_version}}, so that if we upgrade to PHP 7.4 at some point, we'll still be able to make that change in one place and have it apply everywhere.